lib/HttpClient.js

```diff
@@ -223,6 +219,11 @@ function rawRequest(opts, cb) {
         });
 
         emitResult((err || null), req, res);
+
+        // This has to be after res.on('data') in Node >= v10
```

Comment:
nit: It could be useful to add a bit more info about why that is necessary (maybe add the link to the PR in the nodejs repo?).